template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery-validation-unobtrusive/.bower.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "jquery-validation-unobtrusive",
+  "version": "3.2.6",
+  "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",
+  "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",
+  "main": [
+    "jquery.validate.unobtrusive.js"
+  ],
+  "ignore": [
+    "**/.*",
+    "*.json",
+    "*.md",
+    "*.txt",
+    "gulpfile.js"
+  ],
+  "keywords": [
+    "jquery",
+    "asp.net",
+    "mvc",
+    "validation",
+    "unobtrusive"
+  ],
+  "authors": [
+    "Microsoft"
+  ],
+  "license": "http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm",
```

Comment:
Ah, we'll get this fixed in the actual packages. Filed https://github.com/aspnet/jquery-validation-unobtrusive/issues/61.